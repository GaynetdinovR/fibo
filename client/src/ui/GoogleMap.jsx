import styles from "../styles/Ui.module.sass";
import classNames from "classnames";

const GoogleMap = ({ className }) => {
	return (
		<iframe
			src="https://www.google.com/maps/embed?pb=!1m17!1m12!1m3!1d2250.5728539025727!2d37.47757307673972!3d55.66163797305065!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m2!1m1!2zNTXCsDM5JzQxLjkiTiAzN8KwMjgnNDguNSJF!5e0!3m2!1sen!2sru!4v1721155631161!5m2!1sen!2sru"
			className={classNames(className, styles.map)}
			allowFullScreen=""
			loading="lazy"
			referrerPolicy="no-referrer-when-downgrade"
		/>
	);
};

export default GoogleMap;
