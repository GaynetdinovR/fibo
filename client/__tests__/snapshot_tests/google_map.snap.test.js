import renderer from "react-test-renderer";
import { it, expect } from "@jest/globals";
import GoogleMap from "../../src/ui/GoogleMap.jsx";

it("renders correctly", () => {
	const tree = renderer.create(<GoogleMap/>).toJSON();
	expect(tree).toMatchSnapshot();
});
