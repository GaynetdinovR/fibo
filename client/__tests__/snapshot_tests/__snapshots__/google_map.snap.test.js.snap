// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<iframe
  allowFullScreen=""
  className="map"
  loading="lazy"
  referrerPolicy="no-referrer-when-downgrade"
  src="https://www.google.com/maps/embed?pb=!1m17!1m12!1m3!1d2250.5728539025727!2d37.47757307673972!3d55.66163797305065!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m2!1m1!2zNTXCsDM5JzQxLjkiTiAzN8KwMjgnNDguNSJF!5e0!3m2!1sen!2sru!4v1721155631161!5m2!1sen!2sru"
/>
`;
